platform/kube/controller.go

```diff
@@ -391,6 +411,7 @@ func (c *Controller) GetIstioServiceAccounts(hostname string, ports []string) []
 }
 
 func generateServiceAccountID(sa string, ns string, domain string) string {
+	fmt.Printf("%v://%v/ns/%v/sa/%v\n", uriScheme, domain, ns, sa)
```

Comment:
Confused again - why not use the same scheme that is used for service accounts in k8s ? It would look strange, and may make it difficult to migrate the jobs.